src/go/rpk/pkg/cli/acl/acl.go

```diff
@@ -72,17 +72,35 @@ group.
 
 PRINCIPALS
 
-All ACLs require a principal. A principal is composed of two parts: the type
-and the name. Within Redpanda, only one type is supported, "User". The reason
-for the prefix is that a potential future authorizer may add support for
-authorizing by Group or anything else.
+All ACLs require a principal or a role. A principal is composed of two parts:
```

Comment:
It's unclear how a principal and role differ here...
Perhaps:
A principal is comprised of a user and type. Within Redpanda, only the "user" type is supported. The "user" prefix ensures that potential future authorizers can add support for authorization by group or other potential types. All ACLs require a principal or a role.

Reply:
>unclear how a principal and role differ

yeah. it's tricky because, from redpanda's perspective, a role is just a type of principal, but we want to treat them as distinct concepts in docs and at the interface. I think your suggested change expresses that well though.